uui/components/datePickers/DatePickerBody.tsx

```diff
@@ -2,17 +2,21 @@ import * as React from 'react';
 import dayjs, { Dayjs } from 'dayjs';
 import updateLocale from 'dayjs/plugin/updateLocale.js';
 import { cx } from '@epam/uui-core';
-import {
-    MonthSelection, YearSelection, DatePickerBodyBase, DatePickerBodyBaseProps, valueFormat, i18n,
-} from '@epam/uui-components';
+import { MonthSelection, YearSelection, DatePickerBodyBase, DatePickerBodyBaseProps, valueFormat, i18n } from '@epam/uui-components';
 import { DatePickerHeader } from './DatePickerHeader';
 import { Calendar } from './Calendar';
 import css from './DatePickerBody.module.scss';
 
 dayjs.extend(updateLocale);
 
 export interface DatePickerBodyProps extends DatePickerBodyBaseProps<string> {
+    /*
+    * Array of classes for styling the Day.
```

Comment:
Please rework it based on our guidelines - https://github.com/epam/UUI/wiki/Documentation-Guidelines#callbacks
This is the case of Pure function callback

Reply:
fixed